tdi_python/tdiTable.py

```diff
@@ -1957,22 +2013,67 @@ def callback_wrapper(target, handle, cookie):
             entry = self.get_entry(key_content=None, print_entry = False, key_handle=handle)
             callback(dev_id, pipe_id, direction, parser_id, entry)
             return 0
-        return self._cintf.idle_timeout_cb_type(callback_wrapper)
+        return self._cintf.tdi_idle_timeout_cb_type(callback_wrapper)
 
     def set_idle_table_notify_mode(self, enable, callback, interval, max_ttl, min_ttl):
         attr_hdl = self._cintf.handle_type()
-        sts = self._cintf.get_driver().tdi_table_idle_table_attributes_allocate(self._handle, 1, byref(attr_hdl))
+        sts = self._cintf.get_driver().tdi_attributes_allocate(self._handle, self.attributes_type_map(attributes_type_str="IdleTimeout"), byref(attr_hdl))
         if not sts == 0:
             print("idle_table_attributes_allocate failed on table {}. [{}]".format(self.name, self._cintf.err_str(sts)))
             return -1
         self.idle_tmo_callback = self._wrap_idle_timeout_callback(callback)
+        """
         sts = self._cintf.get_driver().tdi_attributes_idle_table_notify_mode_set(attr_hdl,
                                                                                    c_bool(bool(enable)),
                                                                                    self.idle_tmo_callback,
                                                                                    c_uint(interval),
                                                                                    c_uint(max_ttl),
                                                                                    c_uint(min_ttl),
                                                                                    c_void_p(0))
+                                                                                   """
+        sts = self._cintf.get_driver().tdi_attributes_set_value(attr_hdl,
+                0,          # mode
+                c_uint(1)) # notify mode
+        if not sts == 0:
+            print("idle_table_get failed on table {}. [{}]".format(self.name, self._cintf.err_str(sts)))
```

Comment:
set_idle_table_notify_mode: not idle_table_get

Reply:
Look like missing cookie attributes type with tdi_attributes_set_value -- was set at c_void_p(0) - value is 0.